custodial-copy-reconciler/src/main/scala/uk/gov/nationalarchives/reconciler/OcflService.scala

```diff
@@ -55,7 +55,9 @@ object OcflService {
       val chunk = Chunk.from(repo.getObject(ioRef.toHeadVersion).getFiles.asScala).collect {
         case coFile if isNotMetadataFile(coFile.getStorageRelativePath) =>
           val pathAsList = coFile.getStorageRelativePath.split("/")
-          val pathStartingFromRepType = pathAsList.dropWhile(pathPart => !pathPart.startsWith("Preservation_"))
+          val pathStartingFromRepType =
+            if coFile.getStorageRelativePath.contains("/Preservation_") then pathAsList.dropWhile(pathPart => !pathPart.startsWith("Preservation_"))
```

Comment:
Couldn't you just do `pathAsList.dropWhile(pathPart => !pathPart.startsWith("Preservation_") && !pathPart.startsWith("Access_"))`